helios-services/src/test/java/com/spotify/helios/master/OldJobReaperTest.java

```diff
@@ -0,0 +1,175 @@
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.helios.master;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+
+import com.spotify.helios.common.Clock;
+import com.spotify.helios.common.descriptors.Deployment;
+import com.spotify.helios.common.descriptors.Goal;
+import com.spotify.helios.common.descriptors.Job;
+import com.spotify.helios.common.descriptors.JobId;
+import com.spotify.helios.common.descriptors.JobStatus;
+import com.spotify.helios.common.descriptors.TaskStatus;
+import com.spotify.helios.common.descriptors.TaskStatus.State;
+import com.spotify.helios.common.descriptors.TaskStatusEvent;
+
+import org.joda.time.Instant;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptyMap;
+import static java.util.concurrent.TimeUnit.HOURS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.timeout;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class OldJobReaperTest {
+
+  private static final long RETENTION_DAYS = 1;
+  private static final Job DUMMY_JOB = Job.newBuilder().build();
+
+  private static class Datapoint {
+
+    private final Job job;
+    private final List<TaskStatusEvent> history;
+    private final Map<String, Deployment> deployments;
+    private final JobStatus jobStatus;
+    private final boolean expectReap;
+
+    private Datapoint(final String jobName, final Map<String, Deployment> deployments,
+                      final List<TaskStatusEvent> history, final boolean expectReap) {
+      this.job = Job.newBuilder().setName(jobName).build();
+      this.history = ImmutableList.copyOf(history);
+      this.deployments = ImmutableMap.copyOf(deployments);
+      this.jobStatus = JobStatus.newBuilder().setDeployments(this.deployments).build();
+      this.expectReap = expectReap;
+    }
+
+    public Job getJob() {
+      return job;
+    }
+
+    public JobId getJobId() {
+      return job.getId();
+    }
+
+    public List<TaskStatusEvent> getHistory() {
+      return this.history;
+    }
+
+    public Map<String, Deployment> getDeployments() {
+      return this.deployments;
+    }
+
+    public JobStatus getJobStatus() {
+      return this.jobStatus;
+    }
+  }
+
+  private List<TaskStatusEvent> events(final List<Long> timestamps) {
+    final ImmutableList.Builder<TaskStatusEvent> builder = ImmutableList.builder();
+
+    // First sort by timestamps ascending
+    final List<Long> copy = Lists.newArrayList(timestamps);
+    copy.sort((t1, t2) -> {
```

Comment:
I think this is the same as `Collection.sort(copy)` since `Long implements Comparable`